redaxo/src/addons/be_style/plugins/customizer/boot.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * REDAXO customizer.
+ *
+ * Codemirror by : http://codemirror.net/
+ * Marijn Haverbeke <marijnh@gmail.com>
+ */
+
+if (rex::isBackend()) {
+    
+    $config = $this->getConfig();
+
+    rex_view::addCssFile($this->getAssetsUrl('css/styles.css'));
+
+    if ($config['codemirror']) {
+        rex_view::addCssFile($this->getAssetsUrl('vendor/codemirror/codemirror.css'));
+        rex_view::addCssFile($this->getAssetsUrl('vendor/codemirror/theme/'.$config['codemirror_theme'].'.css'));
+        
+        if ($config['codemirror_theme'] != '') {
+            rex_view::setJsProperty('customizer_codemirror_defaulttheme', $config['codemirror_theme']);
+        }
+        
+        rex_view::addJsFile($this->getAssetsUrl('vendor/codemirror/codemirror-compressed.js'));
+    }
+    
+    if ($config['labelcolor'] != '') {
+        rex_view::setJsProperty('customizer_labelcolor', $config['labelcolor']);
+    }
+    
+    if ($config['showlink']) {
+        rex_view::setJsProperty('customizer_showlink', '<h1 class="be-style-customizer-title"><a href="'. rex::getServer() .'" onclick="window.open(this.href); return false">' . rex::getServerName() . '</a><i class="fa fa-external-link"></i></h1>');
```

Comment:
gibt es für diese immer wiederkehrenden mechanismus `onclick="window.open(this.href); return false"` ne css klasse mit der wir diese verhalten via JS aktivieren können, anstatt den code via inline-event handler überall zu duplizieren?

sowas wie `class="new-window"` o.ä. (das JS dazu könnte ja direkt im rex-core sein, da wir auch dort diese art von links verwenden)